teos/api.py

```diff
@@ -84,7 +84,8 @@ def serve(internal_api_endpoint, endpoint, logging_port, min_to_self_delay, auto
     api.logger.info(f"Initialized. Serving at {endpoint}")
 
     if auto_run:
-        wsgi_serve(api.app, listen=endpoint)
+        # Waitress will serve both on IPV4 and IPV6 if localhost is passed as endpoint
```

Comment:
Comment is a bit unclear. Is the purpose of the `replace` to only serve on IPV4 instead?

Reply:
Yeah. If localhost is passed as a param, waitress would serve on two endpoints: `127.0.0.1` and `::1`.

There's no need for it to serve on both, so I changed it to default to IPV4, since I don't think it may be an issue for a local server.

Reply:
Clarified comment in 40399e4